cmd/incus/snapshot.go

```diff
@@ -11,11 +12,12 @@ import (
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 
-	"github.com/lxc/incus/v6/client"
+	incus "github.com/lxc/incus/v6/client"
```

Comment:
That change shouldn't be there

Reply:
ah, my bad, I think `gofmt` does that automatically. Is there a rule you'd suggest to configure it to not do this?

Reply:
gofmt doesn't do that here but goimports has been known to do that sometimes...